build/BuildNuGets.csx

```diff
@@ -151,15 +153,37 @@ int PackFiles(string[] packageNames, string licenseUrl)
         p.StartInfo.FileName = nugetExePath;
         p.StartInfo.Arguments = nugetArgs;
         p.StartInfo.UseShellExecute = false;
+        p.StartInfo.RedirectStandardError = true;
+
+        Console.WriteLine($"{Environment.NewLine}Running: nuget.exe {nugetArgs}");
 
-        Console.WriteLine($"Running: nuget.exe {nugetArgs}");
         p.Start();
         p.WaitForExit();
 
-        if ((exit = p.ExitCode) != 0)
+        var currentExit = p.ExitCode;
+        if (currentExit != 0)
         {
-            break;
+            var stdErr = p.StandardError.ReadToEnd();
+            string message;
+            if (BuildingReleaseNugets && stdErr.Contains("A stable release of a package should not have on a prerelease dependency."))
+            {
+                // If we are building release nugets and if any packages have dependencies on prerelease packages  
+                // then we want to ignore the error and allow the build to succeed.
+                currentExit = 0;
+                message = $"{Environment.NewLine}{file}: {stdErr}";
+            }
+            else
+            {
+                message = $"{Environment.NewLine}{file}: error: {stdErr}";
+            }
+
+            Console.WriteLine(message);
+            File.AppendAllText(ErrorLogFile, message);
```

Comment:
The error log should enable a quick and simple way to go to the folder containing release nugets for any Roslyn drop and check whether / what errors were encountered when building release nugets. This file should also serve as list of the set of release packages that are missing from a given build at any point.